lychee-lib/src/filter/excludes.rs

```diff
@@ -37,54 +36,36 @@ impl Default for Excludes {
 }
 
 impl Excludes {
+    #[inline]
     pub fn regex(&self, input: &str) -> bool {
-        if let Some(excludes) = &self.regex {
-            if excludes.is_match(input) {
-                return true;
-            }
-        }
-        false
+        self.regex.as_ref().map_or(false, |re| re.is_match(input))
     }
 
-    pub fn false_positive(&self, input: &str) -> bool {
-        lazy_static! {
-            static ref FALSE_POSITIVES: RegexSet = RegexSet::new(FALSE_POSITIVE_REGEX).unwrap();
-        }
-        FALSE_POSITIVES.is_match(input)
+    pub fn is_false_positive(input: &str) -> bool {
+        input == FALSE_POSITIVE_PAT[0]
```

Comment:
Even though `FALSE_POSITIVE_PAT` only contains one string so far, I wonder if we still want to iterate through all elements of that slice already so that we don't forget to update this code once we start adding more strings to `FALSE_POSITIVE_PAT`. What do you think?

Reply:
In the future we may want. So this is not future-proof.

But it's really hard to come up with a second 'false-positive' example.
Also, I haven't thought thoroughly but the naming 'false-positive' is not clear enough.

Reply:
True. Let's keep it like that then. Also can't come up with a better name at the moment. Maybe something like `IGNORE_PATTERNS` or `KNOWN_PHONY_URLS` could work. 🤔 We can come up with something better in the future.